FieldOpt/ConstraintMath/README.md

```diff
@@ -4,21 +4,30 @@ The library now only consists of a list of functions that can be called.
 
 The following libraries are needed:
 
-
 - RPolyPlusPlus(https://github.com/sweeneychris/RpolyPlusPlus):
 
 Important note. Later versions of RPoly makes WellIndexCalculation unstable.
 Clone the RPoly repo and then checkout an older commit with the following command:
 
 git checkout c79e0ea87922b5d7624e76f9792f5fd915837c19
 
-
 - Eigen3 library (http://eigen.tuxfamily.org/index.php?title=Main_Page)
 
 RPolyPlusPlus depends on the eigen library. However RPolyPlusPlus links
 to a different directory than the standard path for the Eigen library.
 A workaround to make this link work is to simply copy the Eigen library
-into THIS(write path here) path.
+into the path: /usr/lib/Eigen and the include files into: /usr/include/Eigen.
```

Comment:
I'm not sure if this is actually needed anymore after I added in Alin's `FindEigen3.cmake`. We should test it.